src/reducers/userSlice.js

```diff
@@ -0,0 +1,67 @@
+// import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
```

Comment:
If of no use the please remove

Reply:
Will be adding reducers to this slice. Kept to have the basic boilercode intact